pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+
+
+....................................................
+Before you submit a pull request, please make sure :
+
+- code contains tests relevant for the problem you are solving
+- all new and existing tests passed
+- all your commits have valid informative message
+- changes follow [CMS Naming, Coding, And Style Rules](http://cms-sw.github.io/cms_coding_rules.html)
```

Comment:
I propose to add two lines such as:
>- you are making a request to the correct branch
>- you followed the guide [resolve conflicts & port features](http://cms-sw.github.io/tutorial-resolve-conflicts.html) if this is a backport or if you needed to resolve conflicts (never use `cms-merge-topic` on a PR branch!)

Reply:
> you followed the guide [resolve conflicts & port features](http://cms-sw.github.io/tutorial-resolve-conflicts.html) if this is a backport or if you needed to resolve conflicts (never use `cms-merge-topic` on a PR branch!)

@kpedro88, those are just *your* opinions on how the workflow for making PRs should work, not the only (or the best) way

Reply:
No, it is the best way. Intermediate merges *always* have the potential to cause problems later, and that potential is often realized. Rebasing is simply cleaner, and users who follow the linked guide will have a much easier time.

The rare power user doing a massive private development project can feel free to disregard these instructions, but the average user will certainly benefit from them.